components/match2/wikis/callofduty/brkts_wiki_specific.lua

```diff
@@ -0,0 +1,24 @@
+---
+-- @Liquipedia
+-- wiki=callofduty
+-- page=Module:Brkts/WikiSpecific
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local DateExt = require('Module:Date/Ext')
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+
+local WikiSpecific = Table.copy(Lua.import('Module:Brkts/WikiSpecific/Base', {requireDevIfEnabled = true}))
+
+function WikiSpecific.matchHasDetails(match)
+	return match.dateIsExact
+		or match.date ~= DateExt.epochZero
```

Comment:
will always be true
match.date is iso (e.g. `1970-01-01T00:00:00+00:00`) while DateExt.epochZero is numeric